modules/oe_content_publication/config/install/core.entity_form_display.node.oe_publication.default.yml

```diff
@@ -2,68 +2,94 @@ langcode: en
 status: true
```

Comment:
Could you please put Alternative title, Navigation title, Teaser to the field group  ALTERNATIVE TITLES AND TEASER.

<img width="1078" alt="Screenshot 2020-11-11 at 18 19 25" src="https://user-images.githubusercontent.com/1183951/98829861-bfdc5580-244a-11eb-9eed-f650229c98ac.png">


Such fields like Language, CONTENT OWNER, Redirect link have to be in the end of the form. See another CTs form implementations for example.